src/screens/Customers/EditCustomerMeter.tsx

```diff
@@ -69,19 +70,19 @@ function EditCustomerMeter() {
                                 formik.setFieldValue('meterNumber', '')
                             }
                         }}
-                        label={'Meter Type'}
+                        label={intl(words.x_type, words.meter)}
                         id='meterType'
                         required
                         value={formik.values.meterType}
```

Comment:
because `formik.values.meterType` is one of `Analog Meter` `Smart Meter` or `No Meter` I think it also needs a translation

edit: actually, check the related note in `AddCustomer`, not sure anymore. Translating might break edit / add customer.